Contributing.md

```diff
@@ -82,4 +83,32 @@ to the TC and the TC uses its standard consensus seeking process to evaluate whe
 not to add this new member. Members who do not participate consistently at the level of
 a majority of the other members are expected to resign.
 
-
+# Project Captains
+
+The Express TC can designate captains for individual projects/repos in the
+organizations. These captains are responsible for being the primary
+day-to-day maintainers of the repo on a technical and community front.
+Repo captains are empowered with repo ownership and module publication rights.
+When there are conflicts, especially on topics that effect the Express project
+at large, captains are responsible to bring it up with the TC and drive
+those conflicts to resolution. Captains are also responsible in making sure
+community members follow the community guidelines, maintaining the repo
+and the registry, as well as in providing user support.
+
+To become a captain for a project you will be expected to participate in that
```

Comment:
All of these "you"s need to be re-worded in to the third person from the second person, as the document is meant to be in the third person form.

Reply:
done, disambiguated beyond any doubt: `Like TC members, Repo captains are a subset of committers.`

All the `you`s are replaced with third person